compiler/prelude/prelude.go

```diff
@@ -1,10 +1,8 @@
 package prelude
 
-const Prelude = prelude + numeric + types + goroutines + jsmapping
+//go:generate go run regen.go
```

Comment:
Also, I would suggest renaming `regen.go` to `gen.go` to be more consistent with other Go projects (`gen.go` generates `prelude_min.go`, "regenerates" is not a better description). Or `genmin.go` if you want it to be more specific.

Examples:

- https://github.com/golang/go/blob/master/src/crypto/md5/gen.go
- https://github.com/golang/go/blob/master/src/sort/genzfunc.go
- https://github.com/golang/go/blob/master/src/time/genzabbrs.go

Reply:
Done.